File: Bahasa c/PRAK405-2210817220030-Valencia Angelin Nata.c
#include <stdio.h>
int main(){
    int a, b, hasil, total1, total2;
    hasil = 0; total1 = 0; total2 = 0;
    scanf("%d", &a);
    scanf("%d", &b);
    for(int i = 0; i < a; i++){
        for(int j = i; j >= 0; j--){
            printf("(%d * %d)", j+1, b);
            if(j > 0){
                printf(" + ");
            }
        }
        hasil = (i+1) * b;
        printf(" = %d\n", hasil);
        total1 += hasil;
        total2 += total1;
    }
    printf("%d", total2);
}

File: Bahasa python/PRAK405-2210817220030-Valencia Angelin Nata.py
a, b = input().split()
a = int(a) ; b = int(b)
hasil = 0 ;total1 = 0 ; total2 = 0 ; i = 0
while i < a:
    i += 1 ; j = i
    if i == 0:
        break
    while j > 0:
        j -= 1
        print("(%d * %d)" % (j + 1, b), end=' ')
        if j > 0:
            print(" + ", end=' ')
    hasil = i * b
    total1 += hasil
    print(" = %d" % (total1))
    total2 += total1
print("%d"%(total2))

